frontend/src/views/web/permission-denied.html

```diff
@@ -9,6 +9,13 @@
                     <div class="card-content">
                         <p><strong>Unable to create or select a team.</strong>
                             <br> {{perm.emailError}}</p>
+                        <br>
+                        <div ng-if="!perm.sendMail">
+                            <p> <span class="text-light-black">Didn't get the email?</span> <a class="waves-effect waves-dark dark-link ev-header-overflow" ng-click="perm.requestLink()">Resend Email</a>.</p>
+                        </div>
+                        <div ng-if="perm.sendMail">
+                            <p> <span class="text-light-gray">The verification mail was sent.</span></p>
```

Comment:
@isht3 Please change this message to "The verification mail has been sent to you. Please confirm!"
cc @deshraj

Reply:
@aka-jain Done!